types/enzyme-adapter-react-15/index.d.ts

```diff
@@ -0,0 +1,12 @@
+// Type definitions for enzyme-adapter-react-15 1.0
+// Project: https://github.com/airbnb/enzyme
+// Definitions by: Tanguy Krotoff <https://github.com/tkrotoff>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare class ReactFifteenAdapter {
+}
+
+declare namespace ReactFifteenAdapter {
```

Comment:
Remove the empty namespace, please. We're starting to try to avoid spreading this "hack" in anticipation of tightening our alignment with the both the es6 spec and compatibility with babel's output. If you just use the alternate import syntax it is not required.